bolt/lib/Profile/DataAggregator.cpp

```diff
@@ -1513,9 +1535,11 @@ void DataAggregator::printBranchStacksDiagnostics(
 }
 
 std::error_code DataAggregator::parseBranchEvents() {
-  outs() << "PERF2BOLT: parse branch events...\n";
-  NamedRegionTimer T("parseBranch", "Parsing branch events", TimerGroupName,
-                     TimerGroupDesc, opts::TimeAggregator);
+  std::string BranchEventTypeStr =
+      !opts::ArmSPE ? "branch events" : "SPE branch events in LBR-format";
+  outs() << "PERF2BOLT: " << BranchEventTypeStr << "...\n";
```

Comment:
nit: adding back the verb from the original code:

```suggestion
  outs() << "PERF2BOLT: parse " << BranchEventTypeStr << "...\n";
```